### PR TITLE
910330 [Dev] Updated the Getting Started UG content for the Maui Tools controls.

### PR DESCRIPTION
**Pull request by PrithisIyyappan**
Hi Team, 

I've updated the Getting Started UG content for the controls mentioned below.

- Autocomplete
- Carousel 
- ComboBox 
- MaskedEntry
- NumericEntry
- SfDateTimeSlider
- SignaturePad

Regards,
Prithis I.

### CODE REVIEW

File: MAUI/Autocomplete/Getting-Started.md [hunk 1]
```diff
@@ -7,19 +7,34 @@ control: SfAutocomplete
 documentation: ug
 ---
 
-# Getting Started with .NET MAUI Autocomplete (SfAutocomplete)
+# Getting Started with .NET MAUI Autocomplete
 
-This section explains how to add the Autocomplete control and bind data to it. This section covers only the basic features needed to get started with the Syncfusion [Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control.
+This section guides you through setting up and configuring a [Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) in your .NET MAUI application. Follow the steps below to add a basic [Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) to your project.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Getting-Started.md:12]: Remove the API refernence added in the second place

File: MAUI/Autocomplete/Getting-Started.md [hunk 2]
```diff
@@ -7,19 +7,34 @@ control: SfAutocomplete
...
+
+1. Open Visual Studio or VS Code.
+2. Go to **File > New > Project** and choose the **.NET MAUI App** template.
+3. Name the project and choose a location, then click **Create**.
+
+## Step 2: Install the Syncfusion MAUI Inputs NuGet Package
 
-Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add [.NET MAUI Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Inputs](https://www.nuget.org/packages/Syncfusion.Maui.Inputs) and then install it.
+1. In **Solution Explorer**, right-click the project and choose **Manage NuGet Packages**.
+2. Search for [`Syncfusion.Maui.Inputs`](https://www.nuget.org/packages/Syncfusion.Maui.Inputs) and install the latest version.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Getting-Started.md:34]: remove the highlight

File: MAUI/Autocomplete/Getting-Started.md [hunk 3]
```diff
@@ -7,19 +7,34 @@ control: SfAutocomplete
...
+Ensure the necessary dependencies are installed correctly, and the project is restored.
 
-## Handler registration 
+## Step 3: Handler registration 
 
  In the MauiProgram.cs file, register the handler for Syncfusion core.
```
Comment (naveenkumar-sanjeevirayan) [inline MAUI/Autocomplete/Getting-Started.md:39]: Step - 3 Register the handler
Syncfusion.Maui.Core Nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.

File: MAUI/Autocomplete/Getting-Started.md (hunk 1 above)
Comment (PrithisIyyappan) [reply MAUI/Autocomplete/Getting-Started.md:12]: Removed.

File: MAUI/Autocomplete/Getting-Started.md (hunk 2 above)
Comment (PrithisIyyappan) [reply MAUI/Autocomplete/Getting-Started.md:34]: Removed highlight.

File: MAUI/Autocomplete/Getting-Started.md (hunk 3 above)
Comment (PrithisIyyappan) [reply MAUI/Autocomplete/Getting-Started.md:39]: Added.

File: MAUI/Autocomplete/Getting-Started.md [hunk 4]
```diff
@@ -7,21 +7,36 @@ control: SfAutocomplete
...
+
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here](https://github.com/dotnet/maui/wiki/VS-Code-setup).
 
 To get start quickly with our .NET MAUI Autocomplete, you can check the below video.
 
 {% youtube "https://www.youtube.com/watch?v=ontSh2NZvE4" %}
 
-## Adding a .NET MAUI Autocomplete reference
+## Step 1: Create a New MAUI Project
```
Comment (PaulAndersonS) [inline MAUI/Autocomplete/Getting-Started.md:25]: Step 1: Create a New MAUI Project

Visual Studio

Go to File > New > Project and choose the .NET MAUI App template.
Name the project and choose a location, then click Next.
Select the .NET framework version and click Create.
Visual Studio Code

Open the command palette by pressing `Ctrl+Shift+P`` and type .NET:New Project and enter.
Choose the .NET MAUI App template.
Select the project location, type the project name and press enter.
Then choose Create project

Comment (PrithisIyyappan) [reply MAUI/Autocomplete/Getting-Started.md]: Updated.

File: MAUI/Autocomplete/Getting-Started.md [hunk 5]
```diff
@@ -174,36 +199,45 @@ Now, populate this 'SocialMediaViewModel' data in the [Autocomplete](https://hel
...
+autocomplete.WidthRequest = 250;
+autocomplete.HeightRequest = 50;
+autocomplete.ItemsSource = socialMediaViewModel.SocialMedias;
+Content = autocomplete;
 
 {% endhighlight %}
 {% endtabs %}
 
-N> Set the 'SocialMediaViewModel' instance as the `BindingContext` of your control. This is done to bind the properties of 'SocialMediaViewModel' to the `Autocomplete`.
+N> Set the 'SocialMediaViewModel' instance as the **BindingContext** of your control. This is done to bind the properties of 'SocialMediaViewModel' to the Autocomplete.
```
Comment (Choza-rajan) [inline MAUI/Autocomplete/Getting-Started.md:212]: Set the BindingContext of your page to an instance of SocialMediaViewModel. This allows you to bind the properties of SocialMediaViewModel to the Autocomplete control.

File: MAUI/ComboBox/Getting-Started.md [hunk 6]
```diff
@@ -180,36 +205,45 @@ Now, populate this SocialMediaViewModel data in [ComboBox](https://help.syncfusi
...
+comboBox.WidthRequest = 250;
+comboBox.HeightRequest = 50;
+comboBox.ItemsSource = socialMediaViewModel.SocialMedias;
+Content = comboBox;
 
 {% endhighlight %}
 {% endtabs %}
 
-N> Set the SocialMediaViewModel instance as the BindingContext of your control; this is done to bind properties of SocialMediaViewModel to `ComboBox`.
+N> Set the SocialMediaViewModel instance as the BindingContext of your control; this is done to bind properties of SocialMediaViewModel to ComboBox.
```
Comment (Choza-rajan) [inline MAUI/ComboBox/Getting-Started.md:218]: Set the BindingContext of your page to an instance of SocialMediaViewModel. This allows you to bind the properties of SocialMediaViewModel to the Autocomplete control.

File: MAUI/DateTime-Slider/getting-started.md [hunk 7]
```diff
@@ -7,31 +7,46 @@ control: SfDateTimeSlider
...
+### Visual Studio Code
 
-## Handler registration
+1.  Open the command palette by pressing `Ctrl+Shift+P`` and type **.NET:New Project** and enter.
+2.  Choose the **.NET MAUI App** template.
+3.  Select the project location, type the project name and press enter.
+4.  Then choose **Create project**
 
-In the MauiProgram.cs file, register the handler for the Syncfusion core.
+## Step 2: Install the Syncfusion MAUI Inputs NuGet Package
```
Comment (Choza-rajan) [inline MAUI/DateTime-Slider/getting-started.md:41]: why used input page name in here?

File: MAUI/Masked-Entry/Getting-Started.md [hunk 8]
```diff
@@ -141,7 +165,7 @@ maskedEntry.Mask = "[A-Za-z0-9._%-]+@[A-Za-z0-9]+.[A-Za-z]{2,3}";
 
 ![RegEx mask in .NET MAUI Masked Entry.](maskedentry_images/net-maui-masked-entry-regex-mask.png)
 
-## Setting the prompt character
+## Step 6: Setting the prompt character
```
Comment (Choza-rajan) [inline MAUI/Masked-Entry/Getting-Started.md:168]: remove the steps for this topic. Set the topic directly

File: MAUI/Masked-Entry/Getting-Started.md [hunk 9]
```diff
@@ -168,7 +192,7 @@ maskedEntry.PromptChar = '#';
 
 ![.NET MAUI Masked Entry prompt character.](maskedentry_images/net-maui-masked-entry-prompt-char.png)
 
-## Setting the value
+## Step 7: Setting the value
```
Comment (Choza-rajan) [inline MAUI/Masked-Entry/Getting-Started.md:195]: remove the steps for this topic. Set the topic directly

File: MAUI/SignaturePad/getting-started.md [hunk 10]
```diff
@@ -7,29 +7,45 @@ control: SfSignaturePad
...
-## Handler registration
+### Visual Studio Code
 
-In the MauiProgram.cs file, register the handler for Syncfusion Core.
+1.  Open the command palette by pressing `Ctrl+Shift+P`` and type **.NET:New Project** and enter.
+2.  Choose the **.NET MAUI App** template.
+3.  Select the project location, type the project name and press enter.
+4.  Then choose **Create project**
+
+## Step 2: Install the Syncfusion MAUI Inputs NuGet Package
```
Comment (Choza-rajan) [inline MAUI/SignaturePad/getting-started.md:40]: Wrongly used inputs nuget name in here

File: MAUI/Autocomplete/Getting-Started.md (hunk 5 above)
Comment (PrithisIyyappan) [reply MAUI/Autocomplete/Getting-Started.md]: Updated.

File: MAUI/ComboBox/Getting-Started.md (hunk 6 above)
Comment (PrithisIyyappan) [reply MAUI/ComboBox/Getting-Started.md]: Updated.

File: MAUI/DateTime-Slider/getting-started.md (hunk 7 above)
Comment (PrithisIyyappan) [reply MAUI/DateTime-Slider/getting-started.md]: Addressed.

File: MAUI/Masked-Entry/Getting-Started.md (hunk 8 above)
Comment (PrithisIyyappan) [reply MAUI/Masked-Entry/Getting-Started.md]: Addressed

File: MAUI/Masked-Entry/Getting-Started.md (hunk 9 above)
Comment (PrithisIyyappan) [reply MAUI/Masked-Entry/Getting-Started.md]: Addressed.

File: MAUI/SignaturePad/getting-started.md (hunk 10 above)
Comment (PrithisIyyappan) [reply MAUI/SignaturePad/getting-started.md]: updated

**Review by Choza-rajan — APPROVED** [review]

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**